### DISCUSSION TITLE
Not able to use the GitHub CLI as I have a limit of 0 for GraphQL, which is preventing me from doing anything.  Will upgrading my account to a paid subscriber allow me to use the CLI?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#80297] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Hi, 

I have recently started to try using the GitHub CLI, but I am running into issues with GraphQL rate limits.  Every time I try to do something, I get an error message that says: `GraphQL: API rate limit exceeded for user ID [my user ID]`.  It was really frustrating me, so I checked my rate limits via `https://[api.github.com/rate_limit](https://api.github.com/rate_limit)` just to find that I have zero allowance for GraphQL.  

```
},
"graphql": {
  "limit": 0,
  "remaining": 0,
  "reset": 1702818460,
  "used": 0,
  "resource": "graphql"
},
```

I have a feeling that this might be because I am using a free account, and if I subscribe to a paid account, I might be able to use the GraphQL API.  However, I would like to make sure this is correct.  I have tried searching on like for the API rate limits per subscription tiers for GraphQL, and I have also scoured the documentation provided by GitHub, but I cannot find any information on this and I do not want to subscribe to a paid account if I am not going to be able to use the CLI, as that is the only reason I am considering doing it.  

I do not use GitHub enough to justify having a paid subscription otherwise, but I use it often enough that using the CLI is going to make it more efficient, as I can do everything via the terminal, rather than having to keep switching back and forth between the terminal and the GitHub website.  

So, If I subscribe to a paid account, will that solve this problem of now having any allowances for the GraphQL API, and what level of account will I need?  Is this only available for Business or Enterprise accounts, or will a Pro level personal account be enough?

Thanks.

**Comment by elfin8er** [comment]
I'm having pretty much the same issue. In my case, I have two gh accounts. On this account, I'm getting a GraphQL limit of 5,000 but I have another account which is showing a GraphQL limit of 0. Both accounts are free accounts. The second account is brand new so I'm wondering if there are more GraphQL API limitations for new accounts... ?

I also noticed that the "reset" timer keeps resetting to 1 hour in the future.
